.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[target.aarch64-unknown-linux-gnu]
+linker = "aarch64-linux-gnu-gcc"
```

Comment:
wot's this then eh?

Reply:
Sets the right linker when cross compiling to AArch64.

…this makes me realize: Should I remove this? If you compile from AArch64 to AArch64 the linker has to be just ld, right?

Reply:
I think it'll use gcc even then, but i don't get why specifically this is needed

Reply:
if it doesn't support choosing the right linker when cross-compiling, does it support cross-compiling at all?

Reply:
For some reason it doesn't pick the right linker when you don't specify it explicitly. You know what, I'll back this out for now.